ui/src/views/tools/SelectVmwareVcenter.vue

```diff
@@ -299,18 +300,23 @@ export default {
           this.hosts = response.listvmwaredchostsresponse.host
         }
       }).catch(error => {
-        this.$notifyError(error)
+        if (doNotify) {
+          this.$notifyError(error)
```

Comment:
instead of not notifying the error, can we stop calling the list dc hosts until some pause or enter key pressed?